Cargo.toml

```diff
@@ -17,14 +17,14 @@ members = [
 resolver = "2"
 
 [workspace.dependencies]
-zcash_address = { git = "https://github.com/zingolabs/librustzcash.git", tag = "zcash_client_backend-0.12.1-1-g11ce156b_avoid_wallet_write" } # git tag -s $(git describe --dirty)
-zcash_client_backend = { git = "https://github.com/zingolabs/librustzcash.git", tag = "zcash_client_backend-0.12.1-1-g11ce156b_avoid_wallet_write", features = ["lightwalletd-tonic", "orchard", "transparent-inputs"] }
-zcash_encoding = { git = "https://github.com/zingolabs/librustzcash.git", tag = "zcash_client_backend-0.12.1-1-g11ce156b_avoid_wallet_write" }
-zcash_keys = { git = "https://github.com/zingolabs/librustzcash.git", tag = "zcash_client_backend-0.12.1-1-g11ce156b_avoid_wallet_write", features = ["orchard"] }
+zcash_address = { git = "https://github.com/zingolabs/librustzcash.git", tag = "pyfghdiukxbm" }
```

Comment:
Can you provide a hint for why the pin needed to change for this PR?   Then I could tell at a glance in the manifest *why* the last pin was set.